src/deepsparse/v2/text_generation/nl_engine_operator.py

```diff
@@ -29,7 +29,7 @@
 )
 
 
-__all__ = ["NLEngineOperator"]
+__all__ = ["NLEngineOperator", "NlEngineInput"]
```

Comment:
NL vs Nl?

Reply:
Oh, this is not my diff (just rebased on top of updated `v2`). @dsikka can we standardise this in whichever PR gets merged to `v2` next?